### PR TITLE
Feature serve datamodel

### PR DESCRIPTION
**Pull request by KhaledBousrih**
related to https://github.com/Netsach/concrete-datastore/issues/71

### CODE REVIEW

**Comment by KhaledBousrih** [comment]
new endpoint `/datamodel/` added:

-  `/datamodel/` accepts a query param `data-type` to `json`, will return a json response with the datamodel. if no query `data-format` is given (or a value other than `json`, the response will be a HttpResponse with the yaml in content.
-  `/datamodel/download/` accepts a query param `data-type` to `json`, will dowload the file (if `data-type` is `json`, the file will be a jsob file, otherwise the file is a yaml file

- `/datamodel/viewer/` is a user interface with a possibility to choose between the JSON format and the YAML format. It also gives the user th possibility to copy the content and download it

image

image

File: concrete_datastore/concrete/templates/mainApp/organization-provinces.txt [hunk 1]
```diff
@@ -0,0 +1,462 @@
+100057 ['Bruxelles', 'Brabant Wallon']
```
Comment (lcognat) [inline concrete_datastore/concrete/templates/mainApp/organization-provinces.txt:1]: Shouldn't be there

Comment (KhaledBousrih) [reply concrete_datastore/concrete/templates/mainApp/organization-provinces.txt]: deleted

File: concrete_datastore/routes/urls.py [hunk 2]
```diff
@@ -47,6 +47,11 @@
 urlpatterns = [
     re_path(r'^oauth/', include('social_django.urls', namespace='social')),
     re_path(r'^status/$', service_status_view, name='service-status-view'),
+    re_path(
+        r'^datamodel/(?P<action>download|viewer/?)?$',
```
[the remark this answers is not in the document]
Comment (KhaledBousrih) [reply concrete_datastore/routes/urls.py]: changed

**Pull request merged**